mapillary/controller/image.py

```diff
@@ -342,9 +342,8 @@ def get_image_from_key_controller(key: int, fields: list) -> dict:
     :param fields: The list of fields to be returned
     :type fields: list
 
-    :return: The requested image properties in JSON format
-    # ? should this be A GeoJSON?
-    :rtype: dict
+    :return: The requested image properties in GeoJSON format
+    :rtype: str
```

Comment:
Doesn't it seem strange we're returning an `str` instead of a GeoJSON class object or a `dict` equivalent?

Reply:
Same as [here](https://github.com/facebookincubator/mapillary-python-sdk/pull/90#discussion_r683040355). We need to discuss this further with @cbeddow. And you're right about the `GeoJSON` class object. If we implement the suggestions made by Johan, this will all be returned as an object, we'll need to figure out a proper architecture for that! 😄

Reply:
Noted